datasets/assset/dataset.json

```diff
@@ -0,0 +1,266 @@
+                        "type": "string"
+                    },
+                    "formaat": {
+                        "description": "Aanduiding voor het formaat van elementenverharding.",
+                        "type": "string"
+                    },
+                    "gbdBuurtNaam": {
+                        "description": "Aanduiding van de buurt waarbinnen het beheerobject ligt. (conform CBS-indeling)",
+                        "type": "string"
+                    },
+                    "gbdStadsdeelNaam": {
+                        "description": "Aanduiding van het stadsdeel waarbinnen het beheerobject ligt. Overname uit IMGeo-bestand met woonplaats- of stadsdeelgrenzen.",
+                        "type": "string"
+                    },
+                    "gbdWijkNaam": {
+                        "description": "Aanduiding van de wijk waarbinnen het beheerobject ligt. Overname uit IMGeo-bestand met wijkgrenzen.",
+                        "type": "string"
+                    },
+                    "gebruiksfunctie": {
+                        "description": "Aanduiding van één of meer gebruiksfuncties van het beheerobject.",
+                        "type": "string"
+                    },
+                    "geluidsreducerend": {
+                        "description": "Registratie of een object een geluidsreducerend effect heeft.",
+                        "type": "boolean"
+                    },
+                    "geometrie": {
+                        "$ref": "https://geojson.org/schema/Geometry.json",
+                        "description": "Aanduiding voor de geometrie van het beheerobject volgens het Stelsel van de Rijksdriehoeksmeting (RD, epsg:28992)"
+                    },
+                    "imborVersie": {
+                        "description": "Versie-aanduiding van het Informatiemodel Beheer Openbare Ruimte (IMBOR)",
+                        "type": "string"
+                    },
+                    "jaarLaatsteConservering": {
+                        "description": "Jaar waarin schade(s) aan het beheerobject is geconserveerd.",
+                        "type": "number"
+                    },
+                    "jaarUitgevoerdOnderhoud": {
+                        "description": "Jaar waarin het onderhoud van het beheerobject is uitgevoerd.",
+                        "type": "number"
+                    },
+                    "jaarVanAanleg": {
+                        "description": "Jaar waarin het beheerobject is aangelegd of aangeplant (kan ook voor de plaatsingsdatum gebruikt worden).",
+                        "type": "number"
+                    },
+                    "kleur": {
+                        "description": "Kleur van het beheerobject.",
+                        "type": "string"
+                    },
+                    "lengte": {
+                        "description": "Lengte van het beheerobject.",
```

Comment:
Welke eenheid heeft dit veld? Dit kan in de description maar daarnaast eventueel ook in een 'unit' attribuut: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#permitted-attributes